apps/emqx_dashboard/src/emqx_dashboard_api.erl

```diff
@@ -203,7 +204,9 @@ field(version) ->
 field(old_pwd) ->
     {old_pwd, mk(binary(), #{desc => ?DESC(old_pwd)})};
 field(new_pwd) ->
-    {new_pwd, mk(binary(), #{desc => ?DESC(new_pwd)})}.
+    {new_pwd, mk(binary(), #{desc => ?DESC(new_pwd)})};
+field(role) ->
+    {role, mk(binary(), #{desc => ?DESC(role), example => ?ROLE_DEFAULT})}.
```

Comment:
we'll need to bump api version for this endpoint?
if the old rest-api client calls the same api to create a user, they would expect it to be superuser.

Reply:
no, this change is invisable for open source users, I will change the default role to the same as the old version